examples/apps/simple_wallet/workload/simple_wallet_logic.cpp

```diff
@@ -0,0 +1,179 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include <string.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <vector>
+
+#include "simple_wallet_logic.h"
+#include "simple_wallet_execute_io.h"
+#include "enclave_utils.h"
+
+bool IsValidNumber(std::string numStr) {
```

Comment:
Did you mean IsValidPositiveInteger ? if so rename the function accordingly. Else you could just rename it to IsValidAmount

Reply:
Will rename the method to IsValidAmount.